pkg/client/resmgmt/lifecycleprocessor_test.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
```

Comment:
This needs to be your own copyright - not SecureKey. You can put your company name here or, if you're working independently, your own name.

Reply:
oh, didn't know that, just copy-pasted :) 
I should have done the same to all new files I've created in this SDK and fabric, heh.